lib/routes/qbitai/category.ts

```diff
@@ -31,24 +33,34 @@ export const route: Route = {
 
 async function handler(ctx) {
     const category = ctx.req.param('category');
-    const link = encodeURI(`https://www.qbitai.com/category/${category}/feed`);
-    const feed = await parser.parseURL(link);
+    const url = encodeURI(`https://www.qbitai.com/category/${category}/feed`);
 
-    const items = feed.items.map((item) => ({
+    const feed = await parser.parseURL(url);
+    const entries = feed.items.map((item) => ({
         title: item.title,
         pubDate: parseDate(item.pubDate),
         link: item.link,
         author: '量子位',
         category: item.categories,
-        description: item['content:encoded'],
+        description: '', // Initialize description field
     }));
 
+    await Promise.all(
+        entries.map((entry) => cache.tryGet(entry.link, async () => {
+                try {
+                    const response = await ofetch(entry.link);
+                    const $ = load(response);
+                    entry.description = $('.article').html() || 'No content found';
+                } catch {
+                    entry.description = 'Failed to fetch content';
+                }
+                return entry;
+            }))
+    );
```

Comment:
https://github.com/DIYgod/RSSHub/pull/17740#discussion_r1863723207

> The resolved promises must be passed to `Route['handler']['item']`; otherwise, RSSHub will not cache the response properly.